pkg/storageops/aws/aws.go

```diff
@@ -56,17 +65,21 @@ func NewEnvClient() (storageops.Ops, error) {
 		),
 	)
 
-	return NewEc2Storage(instance, ec2), nil
+	return NewEc2Storage(instance, instanceType, ec2), nil
 }
 
 // NewEc2Storage creates a new aws storage ops instance
-func NewEc2Storage(instance string, ec2 *ec2.EC2) storageops.Ops {
+func NewEc2Storage(instance, instanceType string, ec2 *ec2.EC2) storageops.Ops {
 	return &ec2Ops{
-		instance: instance,
-		ec2:      ec2,
+		instance:     instance,
+		instanceType: instanceType,
+		ec2:          ec2,
 	}
 }
 
+// nvmeInstanceTypes are list of instance types whose EBS volumes are exposed as NVMe block devices
+var nvmeInstanceTypes = []string{"c5", "c5d", "i3.metal", "m5", "m5d", "r5", "r5d", "z1d"}
```

Comment:
this would mean we have to stay updated with the aws instance types

Reply:
Yes that is right and this is the only we can detect a nvme drive.